lib/config.js

```diff
@@ -211,12 +211,9 @@ config.workspaceDevHostHostname = 'localhost';
 config.workspaceHostPort = 8081;
 config.workspaceDevContainerHostname = 'host.docker.internal';
 config.workspaceAuthzCookieMaxAgeMilliseconds = 60 * 1000;
-config.workspaceJobsDirectory = '/jobs/workspaces';
 config.workspaceJobsDirectoryOwnerUid = 0;
 config.workspaceJobsDirectoryOwnerGid = 0;
 config.workspaceJobsParallelLimit = 5;
-config.workspaceMainZipsDirectory = '/workspace_main_zips';
-config.workspaceHostZipsDirectory = '/workspace_host_zips';
```

Comment:
AFAICT there's no reason we can't just use temp files for these, so I opted to do that and eliminate these two config options.